src/Sylius/Behat/Context/Ui/Shop/AccountContext.php

```diff
@@ -243,6 +253,37 @@ public function iShouldBeNotifiedThatThePasswordShouldBeAtLeastCharactersLong()
     }
 
     /**
+     * @When I browse my orders
+     */
+    public function iBrowseMyOrders()
+    {
+        $this->orderIndexPage->open();
+    }
+
+    /**
+     * @Then I should see a single order in the list
+     */
+    public function iShouldSeeASingleOrderInTheList()
+    {
+        Assert::same(
```

Comment:
Why? I think that `Assert::same` is more accurate and it checks if values are identical, so I guess that we should use `::same` instead of `::eq`